src/python/pants/option/global_options.py

```diff
@@ -285,6 +333,13 @@ def from_options(
     remote_execution_overall_deadline_secs=60 * 60,  # one hour
 )
 
+DEFAULT_LOCAL_STORE_OPTIONS = LocalStoreOptions(
+    store_dir=os.path.join(get_pants_cachedir(), "lmdb_store"),
+    processes_max_size_bytes=(4 * GIGABYTES),
+    files_max_size_bytes=(128 * GIGABYTES),
+    directories_max_size_bytes=(4 * GIGABYTES),
```

Comment:
Better is to set these defaults on the dataclass itself, then say `DEFAULT_LOCAL_STORE_OPTIONS = LocalStoreOptions()`.